XU-TechnicalPaper.md

```diff
@@ -78,9 +78,15 @@ As with existing decentralized exchanges, no single point of failure (SPOF) can
 
-Each participating exchange runs its own XU node instance, which opens payment channels on each chain it chooses to support. This means, if an exchange wants to enable its users to trade bitcoin and litecoin through Exchange Union, it has to open a payment channel on the litecoin blockchain AND on the bitcoin blockchain to at least one other exchange in the union. Payment channels in XU are meant to be kept open long-term. In general, all BIP 199 compatible payment channels can be supported directly, others might need to find a way to implement HTLC swaps via smart contracts or similar. The payment channel standard on bitcoin and litecoin is called ‘[lightning](https://github.com/lightningnetwork/lnd)’ and compatible implementations are maintained by the companies [ACINQ](https://github.com/ACINQ/eclair), [Lightning Labs](https://lightning.engineering/) and [Blockstream](https://blockstream.com/). Compatibility was only achieved [recently](https://medium.com/@lightning_network/lightning-protocol-1-0-compatibility-achieved-f9d22b7b19c4), implementation of such is still ongoing. Ethereum’s payment channel technology is called the‘[Raiden Network](https://raiden.network/)’ and is maintained by the company [Brainbot](http://www.brainbot.com/). **Exchange Union will ensure that its XU node implemenatation remains fully compatible with the [lightning protocol specifications (BOLT)](https://github.com/lightningnetwork/lightning-rfc/blob/master/00-introduction.md) and also with the raiden network to support the growth and adoption of payment channels.**
+Each participating exchange runs its own XU node instance, which opens payment channels on each chain it chooses to support. This means, if an exchange wants to enable its users to trade bitcoin and litecoin through Exchange Union, it has to open a payment channel on the litecoin blockchain AND on the bitcoin blockchain to at least one other exchange in the union. Payment channels in XU are meant to be kept open long-term. Also, the current design doesn't encourage high volume transactions in one single channel and most implementations set upper limits as a 'training wheels restriction' (lnd's max. channel size is: 0.16777216 BTC). This is problematic for our use case, since we expect to see high-volume transactions between exchanges, which send transactions to other exchanges on behalf of their entire user base. To achieve this, we will rely on multiple channels, which carry out partial transfers of one atomic payment as proposed in a [new protocol by Laolu and Conner Fromknecht](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-February/000993.html). This allows:
```

Comment:
I don't think I'd call AMP a new "protocol" but rather a new "technique" or perhaps "scheme" which is Laolu's wording from the mailing list.

It's also not particularly difficult to lift the channel size limit (in addition to the channel size limit there is also a payment limit of ~0.043 btc which can also be lifted). The limit is part of version 1.0 of the LN RFC, not of lnd itself. I expect there will be implementations that support larger channels in the not too distant future, or XU could use a modified version of LND that allows XU nodes to have larger channels and payments with each other. It's possible to do that while maintaining backwards compatibility with 1.0 LN nodes.